### ISSUE TITLE
Add `MlflowException.invalid_parameter_value`

### ISSUE THREAD

**Issue by harupy** [mlflow/mlflow#5598]
## Motivation

We often construct an `MlflowException` object with `error_code=INVALID_PARAMETER_VALUE`:

```python
import mlflow.exceptions from MlflowException
from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE

raise MlflowException(
    "error message",
    error_code=INVALID_PARAMETER_VALUE,
)
```

If we had a class method like this:

```
class MlflowException(...):
    @classmethod
    def invalid_parameter_value(cls, message, **kwargs):
        return cls(message, error_code=INVALID_PARAMETER_VALUE, kwargs)
```

We could simplify the original code to:

```
import mlflow.exceptions from MlflowException

raise MlflowException.invalid_parameter_value("error message")
```

**Comment by cterrelljones** [comment]
May I take this one?

**Comment by harupy** [comment]
@cterrelljones Yes, looking forward to your PR!

**Comment by cterrelljones** [comment]
Sorry, this is my first time contributing to open source. I want to push my code but I am getting 

ERROR: Permission to mlflow/mlflow.git denied to cterrelljones.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Is there a step I haven't done yet?

**Comment by harupy** [comment]
@cterrelljones Did you for the repo? https://opensource.com/article/19/7/create-pull-request-github might help.

**Comment by harupy** [comment]
@cterrelljones Yes, you can add `Fix #5598` in https://github.com/mlflow/mlflow/pull/5627 so GitHub will automatically close this issue once #5627 is merged.
